rj_gameplay/rj_gameplay/action/move.py

```diff
@@ -6,20 +6,54 @@
 import stp.action as action
 import stp.rc as rc
 import numpy as np
+from rj_msgs.msg import RobotIntent, MotionCommand, EmptyMotionCommand, PathTargetMotionCommand, LinearMotionInstant
+from rj_geometry_msgs.msg import Point
+from typing import Optional
+import math
+from rj_msgs import msg
 
-class IMove(action.IAction, ABC):
-
-    def done(self) -> bool:
-        pass
-
-class Move(IMove):
+class Move(action.IFiniteAction):
     """
     Move Action
-    TODO: update with actions implementation
     """
-    def __init__(self, point: np.ndarray):
-        self.point = point
+    def __init__(self,
+            robot_id : int,
+            target_point : np.ndarray,
+            target_vel : np.ndarray = np.array([0.0,0.0]),
+            face_angle : Optional[float] = None,
+            face_point : Optional[np.ndarray] = None,
+            priority : int = 0):
+            
+            self.robot_id = robot_id
+            self.target_point = target_point
+            self.target_vel = target_vel
+            self.face_angle = face_angle
+            self.face_point = face_point
```

Comment:
Just a question: what's the difference between face_point and target_point here?